src/pages/mypage/ProfileApplyStatus.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ProfileApplyStatus = ({ applyList }) => {
+  let passCount = 0;
+  let failCount = 0;
+  let documentpassCount = 0;
+
+  for (let i = 0; i < applyList.length; i++) {
+    if (applyList[i].result === '합격') {
+      passCount++;
+    } else if (applyList[i].result === '서류통과') {
+      documentpassCount++;
+    } else {
+      failCount++;
+    }
+  }
```

Comment:
```suggestion
const passCount = applyList.filter(list => list.result === '합격').length;
  const documentpassCount = applyList.filter(list => list.result === '서류통과').length;
  const failCount = applyList.filter(list => list.result === '불합격').length;
```
성주님이 구현하신 방법으로도 가능하고, 필터링 해서 새로운 배열을 반환해서도 카운트해 볼 수 있을 것 같아요!

Reply:
적용해보겠습니다!